server/tests/ballot_polling/conftest.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+from flask.testing import FlaskClient
+
+from ...models import *  # pylint: disable=wildcard-import
+from ..helpers import *  # pylint: disable=wildcard-import
+
+
+@pytest.fixture
+def election_id(client: FlaskClient, org_id: str, request):
+    set_logged_in_user(client, UserType.AUDIT_ADMIN, DEFAULT_AA_EMAIL)
+    return create_election(
+        client,
+        audit_name=f"Test Audit {request.node.name}",
+        audit_type=AuditType.BALLOT_POLLING,
+        audit_math_type=AuditMathType.MINERVA,
+        organization_id=org_id,
+    )
```

Comment:
is there a reason to have a conftest.py here? since you only have one test file (and it's specific to minerva) i think it would be simpler to just move that fixture there

Reply:
Okay, can do. I was trying to preserve as much of the naming as possible so I could be lazy. I was trying to follow the examples of ballot/batch comparison, but I can push it up.